includes/admin/payments/class-payments-table.php

```diff
@@ -502,19 +504,11 @@ public function column_default( $payment, $column_name ) {
 				$value .= sprintf( '<br><small>%1$s %2$s</small>', __( 'via', 'give' ), give_get_gateway_admin_label( $payment->gateway ) );
 				break;
 
-			case 'donation_form':
-				$form_title = empty( $payment->form_title ) ? sprintf( __( 'Untitled (#%s)', 'give' ), $payment->form_id ) : $payment->form_title;
-				$value      = '<a href="' . admin_url( 'post.php?post=' . $payment->form_id . '&action=edit' ) . '">' . $form_title . '</a>';
-				$level      = give_get_donation_form_title(
-					$payment,
-					[
-						'only_level' => true,
-					]
-				);
+			case 'campaign':
 
-				if ( ! empty( $level ) ) {
-					$value .= $level;
-				}
+                $donation = Donation::find($payment->ID);
+
+				$value = '<a href="' . admin_url( 'edit.php?post_type=give_forms&page=give-campaigns&id=' . $donation->campaign->id . '&tab=overview&action=edit' ) . '">' . $donation->campaign->title . '</a>';
```

Comment:
Could we provide a fallback for cases where the donation has no campaign assigned or the campaign doesn’t exist in the database?

Reply:
I'm not sure how that will even happen. Can you think of a scenario?

Reply:
We may be relying on the fact that campaigns cannot be deleted, only archived. That’s partially true, as we only enforce that through the UI, but not against changes an admin could make programmatically or directly in the database.